src/pages/community/reporting_issues.md

```diff
@@ -34,7 +34,7 @@ Please do not disclose the issue to any public forum before working with the sec
 If you have an issue with YuniKorn operation, please follow these guidelines:
 
 If you are having an issue with setup, configuration, or some other form of behavior not matching your expectation, join the user mailing list and ask your questions in that forum.
-See the [Get Involved](get_involved#communication-channels) page for information on mailing lists.
+See the [Get Involved](https://yunikorn.apache.org/community/get_involved#communication-channels) page for information on mailing lists.
```

Comment:
We should use a URL without the base: 
```
[Get Involved](/community/get_involved#communication-channels)
```
That is what we use everywhere else.

Reply:
I will review all the URLs and update it